pom.xml

```diff
@@ -22,22 +22,22 @@
    This software consists of voluntary contributions made by many
    individuals on behalf of the Apache Software Foundation.  For more
    information on the Apache Software Foundation, please see
-   <http://www.apache.org/>.
+   <https://www.apache.org/>.
```

Comment:
@michael-o let's revert changes to the ASL header and move on.

Reply:
Alright, will do.